### PR TITLE
Commands/button to toggle plot pane status

### PR DESCRIPTION
**Pull request by pfitzseb**
also show all plot pane related commands in the command palette

Fixes https://github.com/julia-vscode/julia-vscode/issues/2330.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -417,32 +437,6 @@
...
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-last"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-previous"
-                }
-            ],
```
Comment (davidanthoff) [inline package.json:445]: Obviously not part of this PR, but what happened to all of these commands? Or are they still here and this is just removing a condition?

**Review by davidanthoff — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (pfitzseb) [reply package.json:445]: Yeah, this just unhides them in the command pane.
